Integration tests fail

**Describe the bug**
Integration tests fail.

**How to reproduce**
Steps to reproduce the behavior:
- Clone repo
- Checkout main branch
- On your local machine set all required env vars (https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/CONTRIBUTING.md#integration-tests)
- Run:
```
make integration-tests
```

**Expected behavior**
Integration tests pass.

**Log output**
```
...
2022-04-14 13:12:16 DEBUG 		Applying config `My SLO_1649934717623147000AllConfigs` using test-resources/integration-all-configs/project/slo/slo.json
2022-04-14 13:12:16 INFO  Deployment summary:
2022-04-14 13:12:16 ERROR Deployment to environment1 failed with error!
2022-04-14 13:12:16 ERROR 	Failed to create DT object My SLO_1649934717623147000AllConfigs (HTTP 400)!
    Response was: {"error":{"code":400,"message":"Validation failed for 1 Validators.","constraintViolations":[{"path":"builtin:monitoring.slo/0","message":"Cannot apply entity selector to metric expression.","parameterLocation":"PAYLOAD_BODY","location":null}]}}, responsible config: test-resources/integration-all-configs/project/slo/slo.json
2022-04-14 13:12:16 ERROR Errors during deployment! Check log!
    all_configs_integration_test.go:45: assertion failed: 1 (statusCode int) != 0 (int)
--- FAIL: TestIntegrationAllConfigs (19.21s)
...
```

**Additional context**
_slo.json_ at test is filtering for type(\"HOST\"), however metric is of type "builtin:service....".

--- a/cmd/monaco/test-resources/integration-all-configs/project/slo/slo.json
+++ b/cmd/monaco/test-resources/integration-all-configs/project/slo/slo.json
@@ -6,7 +6,7 @@
   "metricNumerator": "builtin:service.errors.server.successCount",
   "metricDenominator": "builtin:service.requestCount.total",
   "evaluationType": "AGGREGATE",
-  "filter": "mzName(\"{{ .mzone }}\"),type(\"HOST\")",
+  "filter": "mzName(\"{{ .mzone }}\"),type(\"SERVICE\")",
   "target": 95,
   "warning": 97.5,
   "timeframe": "-1d"

